public/javascripts/table_anim.js

```diff
@@ -1655,6 +1690,12 @@ anlixDocumentReady.add(function() {
           if (device.mesh_master !== '' && device.mesh_master !== undefined) {
             // Skip mesh slaves, master draws their form
             continue;
+          } else if (device.mesh_mode === 10) {
+            device.mesh_slaves = res.devices.filter(
+              (d) => d.mesh_master === device._id,
+            ).map(
+              (d) => d._id,
+            );
```

Comment:
Acho que assim ficaria mais fácil de entender:
```suggestion

          // Fill `mesh_slaves` of the device if it is in vendor mesh mode
          } else if (device.mesh_mode === 10) {
            device.mesh_slaves = res.devices.filter(
              (repeater) => repeater.mesh_master === device._id,
            ).map(
              (repeater) => repeater._id,
            );
```